manifests/rediscluster.pp

```diff
@@ -29,9 +32,11 @@
       ensure  => present,
       content => template('sunet/rediscluster/server.conf.erb'),
     }
-    -> sunet::misc::ufw_allow { "redis_port_${i}":
```

Comment:
Should this be wrapped in:

```
if $::facts['sunet_nftables_enabled'] == 'yes'  or $::facts['dockerhost_advanced_network'] == 'yes' {
....
}
```

Leaving the old ufw rules inplace if those conditions are not met?

Reply:
Good catch, fixed!